fhttp/http_client.go

```diff
@@ -443,8 +445,10 @@ func (c *Client) Fetch() (int, []byte, int) {
 }
 
 // GetIPAddress get the ip address that DNS resolves to when using stdClient.
-func (c *Client) GetIPAddress() string {
-	return c.req.RemoteAddr
+func (c *Client) GetIPAddress() (*stats.Occurrence, int) {
+	occurrence := stats.NewOccurrence()
+	occurrence.Record(c.req.RemoteAddr)
```

Comment:
maybe fine for this iteration but couldn't we record in std client as well?

Reply:
Updated